Source/RapyutaSimulationPlugins/Private/Tools/ROS2Spawnable.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Created by lng on 2021/10/13.
+//
+
+#include "Tools/ROS2Spawnable.h"
+
+// rclUE
+#include "Srvs/ROS2SpawnEntitySrv.h"
+
+void UROS2Spawnable::InitializeParameters(FROSSpawnEntity_Request Request)
+{
+    try{
+        SetName(Request.state_name);
+    } catch (const std::exception& e){
+        UE_LOG(LogTemp, Warning, TEXT("No state_name"));
+    }
+    try{
+        SetNamespace(Request.robot_namespace);
+    } catch (const std::exception& e){
+        UE_LOG(LogTemp, Warning, TEXT("No robot_namespace"));
+    }
```

Comment:
* We now have `LogRapyutaCore` as defined [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/main/Source/RapyutaSimulationPlugins/Private/RapyutaSimulationPlugins.cpp#L19), so it could be used I guess.
* Since both of the name setting functions look quite straightforward, not sure which exceptions could happen tho?

Reply:
Ahh i see, ill take a look at the LogRapyutaCore. I set these exceptions thinking that there maybe a case in the future where a changed spawnEntity message or the entity being spawned might not need to use all the parameters, and having these parameters being set as null might cause issues